src/formatDate.js

```diff
@@ -50,7 +50,52 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorForNewFormat = toFormat.slice(-1);
+  const newDate = date.split(fromFormat[3]);
+  const newArr = [];
+
+  let day, month, year, lengthYear;
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    if (fromFormat[i].includes('Y')) {
+      year = newDate[i];
+    }
+
+    if (fromFormat[i].includes('M')) {
+      month = newDate[i];
+    }
+
+    if (fromFormat[i].includes('D')) {
+      day = newDate[i];
+    }
```

Comment:
it's better to overwrite it with switch case statement as it will make a code cleaner